docs/running-nio/README.md

```diff
@@ -61,6 +61,12 @@ Once your instance is loaded and available, you can add services and blocks in t
 
 Available nio Blocks can be explored in the nio [Block Library](https://blocks.n.io) where you will find a summary of the block's purpose, a list of its properties, commands, inputs, and outputs, and a link to the block code repository.
 
+> **[info] Long-Running**
```

Comment:
There weren't any other references to `long-running.md`, and this seemed like a good place for one.